third_party/closure-compiler/src/test/com/google/javascript/jscomp/RewriteJsonToModuleTest.java

```diff
@@ -159,7 +159,7 @@ public void testPackageJsonBrowserFieldAdvancedUsageGH2625() {
                 lines(
                     "{ 'main': 'foo/bar/baz.js',",
                     "  'browser': { './a/b.js': './c/d.js',",
-                    "               './server.js': 'client.js'} }"))),
```

Comment:
This shouldn't be changed - it's just a coincidence that this third-party test refers to files with the same names.

So all changes under /closure-compiler/ in this PR need to be reverted.